pkg/jwt/persistent/persistent.go

```diff
@@ -294,6 +294,11 @@ func (t *TokenService) DecodeToken(ctx context.Context, token string) (*TokenCon
 }
 
 func (t *TokenService) NewToken(ctx context.Context, context TokenContext) (string, error) {
+	var picture string
+	if !strings.HasPrefix(picture, "data:") {
+		// Don't store the picture in the token if it is a base64 encoded image
+		picture = context.Picture
+	}
```

Comment:
This doesn't seem right? We declare picture, then check it (when it will always be empty), and never use it.